### ISSUE TITLE
InvalidOperationException still present

### ISSUE THREAD

**Issue by Allabreve** [voxeltycoon/bug-tracker#146]
**Describe the bug**
Ware icons not visible/ stuck at top of the screen after loading my save game.
Log output:

```
InvalidOperationException: Can't get vehicle type from location with no vehicles serviced.
  at VoxelTycoon.Tracks.VehicleStationLocationManager.GetServicedVehicleType (VoxelTycoon.Tracks.VehicleStationLocation location) [0x0003e] in <a279c0ce20404b46947f97a3088aa547>:0 
  at VoxelTycoon.Game.UI.VehicleStationLocationIndicator.InvalidateContent () [0x00005] in <a279c0ce20404b46947f97a3088aa547>:0 
  at VoxelTycoon.UI.Indicator.OnLateUpdate () [0x00139] in <a279c0ce20404b46947f97a3088aa547>:0 
  at VoxelTycoon.UI.IndicatorManager.OnLateUpdate () [0x0003e] in <a279c0ce20404b46947f97a3088aa547>:0 
  at VoxelTycoon.UpdateBehaviour.LateUpdate () [0x0000a] in <a279c0ce20404b46947f97a3088aa547>:0 
 
(Filename: <a279c0ce20404b46947f97a3088aa547> Line: 0)
```
**Steps to reproduce**
 - Start a new game
 - Create two Terminals
 - Setup a route
 - Delete one of the Terminals
 - Create a new Terminal at a different location
 - Update your route
 - Save game
 - Reload game from main menu

**Build version**
0.76 2019-08-20

**Platform**
OSX 10.14.6

**Reproducibility**
every time

Maybe related: #145

**Comment by andrewpey** [comment]
Thanks for reporting it, we'll take a look.

**Comment by andrewpey** [comment]
The fix is deployed with `0.76 2019-08-23`.
